api/v1/auth.go

```diff
@@ -40,7 +40,7 @@ func AuthMiddleware(next echo.HandlerFunc) echo.HandlerFunc {
 			return c.JSON(401, err.Error())
 		}
 
-		if !res.Validated {
+		if !res.Validated || res.Meta.TenantID == 0 {
```

Comment:
i don't think we need this check for `== 0` here, when we wire it up to the database, it should either return a valid tenant or not - we should not need to do any comparisons on the ID thats returned for it